packages/playground/_stories/main/Input/Input.stories.ts

```diff
@@ -99,6 +99,16 @@ WithSuggestions.args = {
 	showClearIcon: true
 };
 
+export const SuggestionsWrapping = Template.bind({});
+SuggestionsWrapping.args = {
+	default: `<ui5-suggestion-item text="Wireless DSL/ Repeater and Print Server Lorem ipsum dolar st amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor incidunt ut labore et dolore magna aliquyam erat, diam nonumy eirmod tempor individunt ut labore et dolore magna aliquyam erat, sed justo et ea rebum."></ui5-suggestion-item>
+	<ui5-suggestion-item text="Widescreen Portable DVD Player w MP3, consetetur sadipscing, sed diam nonumy eirmod tempor invidunt ut labore et dolore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergen, no sea takimata. Tortor pretium viverra suspendisse potenti nullam. Congue quisque egestas diam in arcu cursus.Rutrum tellus pellentesque eu tincidunt tortor. Nec tincidunt praesent semper feugiat nibh sed"></ui5-suggestion-item>
+	<ui5-suggestion-item text="Portable DVD Player with 9 inches LCD Monitor"></ui5-suggestion-item>`,
+	placeholder: "Enter product",
+	showSuggestions: true,
+};
+
+
 export const WithValueStateMessage = Template.bind({});
```

Comment:
I think it was discussed with the team to remove "With" prefix from all samples.

Reply:
This will be handled in another change for refactoring